opendevin/action/agent.py

```diff
@@ -32,7 +33,8 @@ def run(self, controller: "AgentController") -> "Observation":
 
     @property
     def message(self) -> str:
-        return f"Thinking: {self.thought}"
+        return f"I'm pondering over the idea: '{self.thought}'. Give me a second to process this."
```

Comment:
Let's just make this `self.thought`
```suggestion
        return self.thought
```

Reply:
actually--this should probably just be `self.content`, and we should remove `thought` as a variable